src/Hydra/Serializer/PartialCollectionViewNormalizer.php

```diff
@@ -81,41 +81,20 @@ public function normalize($object, $format = null, array $context = [])
             return $data;
         }
 
+        $cursorPaginationAttribute = null;
         $metadata = isset($context['resource_class']) && null !== $this->resourceMetadataFactory ? $this->resourceMetadataFactory->create($context['resource_class']) : null;
         $isPaginatedWithCursor = $paginated && null !== $metadata && null !== $cursorPaginationAttribute = $metadata->getCollectionOperationAttribute($context['collection_operation_name'] ?? $context['subresource_operation_name'], 'pagination_via_cursor', null, true);
 
-        $data['hydra:view'] = [
-            '@id' => IriHelper::createIri($parsed['parts'], $parsed['parameters'], $this->pageParameterName, $paginated && !$isPaginatedWithCursor ? $currentPage : null),
-            '@type' => 'hydra:PartialCollectionView',
-        ];
+        $data['hydra:view'] = ['@id' => null, '@type' => 'hydra:PartialCollectionView'];
```

Comment:
I don't think that `@id` should be null, maybe remove the key completely?

Reply:
I agree, this is what I did but I finally changed to preserve the order of the properties.